modules/oe_list_pages_open_vocabularies/tests/src/FunctionalJavascript/OpenVocabulariesFiltersTest.php

```diff
@@ -0,0 +1,370 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_list_pages_open_vocabularies\FunctionalJavascript;
+
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\node\Entity\Node;
+use Drupal\node\NodeInterface;
+use Drupal\oe_list_pages\FilterConfigurationFormBuilderBase;
+use Drupal\open_vocabularies\Entity\OpenVocabulary;
+use Drupal\open_vocabularies\Entity\OpenVocabularyAssociation;
+use Drupal\search_api\Entity\Index;
+use Drupal\taxonomy\Entity\Term;
+use Drupal\Tests\oe_link_lists\Traits\LinkListTestTrait;
+use Drupal\Tests\oe_list_pages\FunctionalJavascript\ListPagePluginFormTestBase;
+
+/**
+ * Tests the List pages open vocabularies filters.
+ *
+ * @group oe_list_pages
+ */
+class OpenVocabulariesFiltersTest extends ListPagePluginFormTestBase {
+
+  use LinkListTestTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'block',
+    'options',
+    'facets',
+    'entity_reference_revisions',
+    'oe_list_pages',
+    'oe_list_page_content_type',
+    'oe_list_pages_filters_test',
+    'oe_list_pages_open_vocabularies_test',
+    'oe_list_pages_link_list_source',
+    'open_vocabularies',
+    'node',
+    'emr',
+    'emr_node',
+    'search_api',
+    'search_api_db',
+  ];
+  /**
+   * The association to test.
+   *
+   * @var \Drupal\open_vocabularies\OpenVocabularyAssociationInterface
+   */
+  protected $association;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    /** @var \Drupal\emr\EntityMetaRelationInstaller $installer */
+    $installer = \Drupal::service('emr.installer');
+    $installer->installEntityMetaTypeOnContentEntityType('oe_list_page', 'node', [
```

Comment:
This is not needed as it gets installed when installing the `oe_list_page_content_type ` module.